#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_bonus_control_code_in_clients_conf_5.dpatch by Paul Hampson <Paul.Hampson@anu.edu.au>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: An extraneous ^g in the manpage source caused '.PP' to appear in the page

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./man/man5/clients.conf.5 /tmp/dpep-work.aqmpOw/freeradius-1.0.4/man/man5/clients.conf.5
--- ./man/man5/clients.conf.5	2005-06-16 19:18:08.000000000 +1000
+++ /tmp/dpep-work.aqmpOw/freeradius-1.0.4/man/man5/clients.conf.5	2005-06-22 17:39:09.920463309 +1000
@@ -36,7 +36,7 @@
 .B client
 section are listed below. Required attributes are labelled as
 such. All other attributes are optional.
-.PP
+.PP
 .TP 0.5i
 .B secret [Required]
 The RADIUS shared secret used for communication between the client/NAS
